src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -352,6 +363,7 @@ public String toString() {
                         .put("cropMarker", this.cropMarker)
                         .put("highlightPreTag", this.highlightPreTag)
                         .put("highlightPostTag", this.highlightPostTag)
+                        .put("matchingStrategy", this.matchingStrategy.toString())
```

Comment:
Maybe now you don't need the `toString()` call here since you will get a null or string btw, unless the null can break it.